src/arrayMethodSort.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
```

Comment:
The condition in the `if` statement should explicitly check for a value greater than 0. While the current code works due to type coercion, being explicit with your conditions can improve code readability and prevent potential bugs in more complex scenarios.